apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -59,10 +62,11 @@
 {{/radio}}
 {{^radio}}
 <!-- Weirdly enough putting this in a div causes it to display without the inline -->
-<span
+<fieldset
```

Comment:
Is a fieldset needed here? This is for dropdown mode, which renders a `<select>`.